test/functional/README.md

```diff
@@ -96,32 +96,33 @@ the Bitcoin Core node application logic. For custom behaviour, subclass the
 P2PInterface object and override the callback methods.
 
 - Can be used to write tests where specific P2P protocol behavior is tested.
-Examples tests are `p2p_unrequested_blocks.py`, `p2p_compactblocks.py`.
+Examples tests are [p2p_unrequested_blocks.py](p2p_unrequested_blocks.py), [p2p_compactblocks.py](p2p_compactblocks.py).
 
-### test-framework modules
+### Test framework modules
+The following are some useful modules for test developers. They are all located in [test/functional/test_framework](test_framework/) 
```

Comment:
Missing punctuation at end of the line. For brevity can omit "some" and "all". Would it be better to add a slash at the end of the `test/functional/test_framework/` label?

Reply:
(Ideally limit line lengths to 120 chars)

Reply:
Thanks for your feedback. Fixed.